### ISSUE TITLE
Testing the data distillery

### ISSUE THREAD

**Issue by ReneRanzinger** [glygener/glygen-issues#469]
Jeet, René:

As I mentioned during the meeting, we now have a distribution of Data Distillery available. The goal is a solution that allows you to set up a local instance of Data Distillery on a laptop. You will be able to connect to your local instance of neo4j and work directly in Cypher.

This [Zip file](https://g-24f5cc.09193a.5898.dn.glob.us/projects/data-distillery/Validated/Distribution/DataDistillery11June2023.zip) in Globus contains a bash script and a folder that contains a set of CSV files. 

* Pre-requisite: Docker Desktop
* Download the Zip file from Globus.
* Expand the Zip file.
* Go into the root folder of the expanded folder named DataDistillery11June2023.
* Run the script named run.sh with parameters for password, neo4j browser port, and neo4j bolt port.

The Zip contains a README.md file that explains the process. Look to the "Quick Start Guide".

We're still refining the documentation to make the process as simple as possible. I'm not sure if it can get much simpler than this, though.

Regards,

J. Alan Simmons

**Comment by ReneRanzinger** [comment]
HI,

thanks for sharing this. I tried to access the file with my university account (rr@uga.edu). When I tried to sign in it ask me to give Hubmap permission to do some things with me data (associated with account but also to show it). I could not deselect individual options so I decided to deny all. Which ended me on the screen in the screenshot.

When I try re-login it don't even get to the place anymore but it tells me "The identities (rr@uga.edu) have not been granted access to this collection. You must link with an identity which has been granted access to this collection."

Best,
  René
